docs/README.md

```diff
@@ -333,15 +333,15 @@ The following options are supported by all filters except `Callback` and `Finder
   processing of the filter will be skipped. If it returns `array` it will be used
   as filter arguments.
   
-```php
-// PostsTable::initialize()
-    $searchManager->like('q', [
-        'fields' => ['Posts.title', 'Authors.title'],
-        'beforeProcess' => function ($query, $args) {
-            $query->contain('Authors');
-        },
-    ]);
-```
+    ```php
+    // PostsTable::initialize()
+        $searchManager->like('q', [
+            'fields' => ['Posts.title', 'Authors.title'],
+            'beforeProcess' => function (Query $query, array $args) {
```

Comment:
If you want to add typehint then use FQCN else copy paste artists / newbies won't add "use" statement and open frivolous issues.

Reply:
Fair enough